### ISSUE TITLE
sqlite: Any plans to add sqlite-typed?

### ISSUE THREAD

**Issue by gmaclennan** [ForbesLindesay/atdatabases#264]
This is a really great tool, and great to see the [new implementation](https://github.com/ForbesLindesay/atdatabases/pull/260) on better-sqlite3.

Any plans to create a `sqlite-typed` module similar to those for pg and mySql? Or would you be open to contributions to add this?

**Comment by ForbesLindesay** [comment]
Yes, I'd absolutely be open to this being added. It is on my list of ideas to build eventually, but if you'd like to build it and submit a PR, that would be great, and likely mean it's available much sooner.

**Comment by neyl66** [comment]
Hello, has there been any progress?
